resources/views/partials/navbar.blade.php

```diff
@@ -22,6 +22,12 @@ class="inline-flex items-center justify-center p-2 rounded-md text-white hover:t
               aria-current="page">
               Home
             </a>
+            @if (Auth()->user() && Auth()->user()->role->value === "administrator")
+                <a href="{{ route('admin.users') }}"
+                class="text-white hover:bg-indigo-600 hover:text-white px-3 py-2 rounded-md text-sm font-medium">
+                Admin
+                </a>
+            @endif
```

Comment:
Look at this [package](https://spatie.be/docs/laravel-permission/v5/introduction), I think should be better for roles and permissions cases instead of creating own system.